pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -147,6 +147,12 @@ func (service serviceMetadata) match(rule *rbacproto.AccessRule) bool {
 	}
 
 	// Check if the constraints are matched.
+	return service.areConstraintsMatched(rule)
+}
+
+// areConstraintsMatched returns True if the calling service's attributes and/or labels match to
+// the ServiceRole constraints.
+func (service serviceMetadata) areConstraintsMatched(rule *rbacproto.AccessRule) bool {
```

Comment:
Maybe just `areConstraintsMatched` -> `matchConstraints`

Reply:
I think generally functions that return bool should have a verb like is, are, should, can, etc... to indicate that they either return `true` for `false`. Wdyt?